src/app/shared/components/list/list.service.ts

```diff
@@ -6,6 +6,7 @@ import { Subject } from 'rxjs/Subject';
 @Injectable()
 export class ListService {
   public onAction = new Subject<void>();
+  public onCreation = new Subject<any>();
```

Comment:
Please remove this, try to use irrespective service for that.
Please merge all the changes from this branch https://github.com/bwsw/cloudstack-ui/tree/list-service-remove